.github/workflows/ci.yml

```
@@ -58,7 +58,7 @@ jobs:
         run: |
           git clone --depth 1 https://github.com/github/gitignore.git
```

what about something like:

```suggestion
          git clone --depth 1 https://github.com/github/gitignore.git --revision b19bb58983cdae1e3a3fd6eafba0b9f2ec3c53ca
```

to avoid future similar issues

---

Yes! great idea